LocalSettings.php

```diff
@@ -506,6 +506,7 @@
 			"$IP/extensions/FlaggedRevs/backend/schema/mysql/FlaggedRevs.sql",
 			"$IP/extensions/Flow/flow.sql",
 			"$IP/extensions/GlobalBlocking/localdb_patches/setup-global_block_whitelist.sql",
+			"$IP/extensions/GlobalPreferences/sql/tables.sql",
```

Comment:
This isn’t something we want on all wikis.

The table shouldn’t even be in any wiki.